frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ScheduleChangeRequestDialog.tsx

```diff
@@ -0,0 +1,106 @@
+
+export interface ScheduleChangeRequestDialogProps {
+    title: string;
+    primaryButtonText: string;
+    open: boolean;
+    onConfirm: (selectedDate: Date) => void;
+    onClose: () => void;
+    projectId: string;
+    environment: string;
+    disabled?: boolean;
+    scheduledAt?: string;
+}
+
+const StyledContainer = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: theme.spacing(2),
+}));
+
+export const ScheduleChangeRequestDialog: FC<ScheduleChangeRequestDialogProps> =
+    ({
+        open,
+        onConfirm,
+        onClose,
+        title,
+        primaryButtonText,
+        projectId,
+        environment,
+        disabled,
+        scheduledAt,
+    }) => {
+        const [selectedDate, setSelectedDate] = useState(
+            scheduledAt ? new Date(scheduledAt) : new Date(),
+        );
+        const [error, setError] = useState<string | undefined>(undefined);
+
+        const timezone = getBrowserTimezoneInHumanReadableUTCOffset();
+
+        return (
+            <Dialogue
+                title={title}
+                primaryButtonText={primaryButtonText}
+                secondaryButtonText='Cancel'
+                open={open}
+                onClose={onClose}
+                onClick={() => onConfirm(selectedDate)}
+                permissionButton={
+                    <PermissionButton
+                        variant='contained'
+                        onClick={() => onConfirm(selectedDate)}
+                        projectId={projectId}
+                        permission={APPLY_CHANGE_REQUEST}
+                        environmentId={environment}
+                        disabled={disabled}
+                    >
+                        {primaryButtonText}
+                    </PermissionButton>
+                }
+                fullWidth
+            >
+                <Alert
+                    severity={'info'}
+                    sx={{ mb: (theme) => theme.spacing(2) }}
+                >
+                    Time zone is not supported. Message about that the time is
+                    based on the viewer browser.
+                </Alert>
+                <Typography
+                    variant={'body1'}
+                    sx={{ mb: (theme) => theme.spacing(4) }}
+                >
+                    Select the date and time when these changes to be applied.
+                    You will be able to modify the selected time or even to
+                    apply directly the changes if needed.
+                </Typography>
+                <StyledContainer>
+                    <DateTimePicker
+                        label='Date'
+                        value={selectedDate}
+                        onChange={(date) => {
+                            setError(undefined);
+                            if (date < new Date()) {
+                                setError('Invalid date, must be in the future');
```

Comment:
Can we make this a little clearer? 

```suggestion
                                setError(`The time you provided (${date.toLocaleString()}) is not valid because it's in the past. Please select a time in the future.`);
```

Optionally, we can remove the interpolation of the provided date.